electrumx/lib/tx.py

```diff
@@ -39,6 +39,7 @@
     unpack_le_uint32_from, unpack_le_uint64_from, pack_le_int32, pack_varint,
     pack_le_uint16, pack_le_uint32, pack_le_int64, pack_varbytes,
 )
+from electrumx.server.mempool import SkipTxDeserialize
```

Comment:
modules in `lib/` should usually not import from modules in `server/`, but the other way around.
so could you move the definition to here, and import it in `mempool.py`?

Reply:
Yeah that seemed weird.  They comment at https://github.com/spesmilo/electrumx/pull/196#discussion_r996188675 suggested to define it in mempool, but I probably misunderstood.  I'll move it.